search_radius: 30.

rrt_iterations: 300

dilation_radius: 0.4

occupancy_threshold: 50
p_uniform: 0.2
p_gaussian: 0.2
bridge_std_dev: 1.
gaussian_std_dev: 5.
p_reverse: 0.5
